src/services/kratos/identity.ts

```diff
@@ -63,6 +63,18 @@ export const IdentityRepository = (): IIdentityRepository => {
         hasNext = page !== undefined && totalCount !== 0
       }
 
+      // * fixes bug in kratos res.headers.link not returning last page properly
+      const pageCount = totalCount / perPage
+      const isPageCountWholeNumber = pageCount % 1 // 0 means its a whole number, therefore, we can skip because no partial last page of data exists
+      if (isPageCountWholeNumber !== 0) {
+        const lastPageNum = Math.ceil(totalCount / perPage)
```

Comment:
check page description (is not sequential) https://www.ory.sh/docs/kratos/reference/api#tag/identity/operation/listIdentities so not sure if this will fix the issue

Reply:
yup. although it's on the description not sure if this is the case in practice... I don't have a lot more context @ntheile than what @dolcalmi said, so feel free to try it if you think this is worth it.

Reply:
@dolcalmi yeah the documentation does not match how to works in practice (maybe they have not updated the documentation yet). The `request.headers.link` has many bugs (it's actually not supported in some browsers). Here is a thread on kratos slack about the topic https://ory-community.slack.com/archives/C012RJ2MQ1H/p1673618866406469